### DISCUSSION TITLE
能否将解析的规则改变（或能让用户改变）一下？

### DISCUSSION THREAD

**Discussion by xuanxuan1231** [jeessy2/ddns-go#1092] [Q&A]
如题，我目前使用的时候会发现ddns-go会将获取到的IP直接添加，导致我的IP过期或重置之后变化，而解析记录仍然存在。因此想问一下能否让我将添加改为替换？

**Comment by xuanxuan1231** [comment]
也有可能本来是有的但是我不知道，所以还请告诉一下，谢谢

**Comment by jeessy2** [comment]
自动的。自行测下就知道了

**Reply by xuanxuan1231** [reply]
行，那我再测试一下，感谢

**Comment by xuanxuan1231** [comment]
刚才看了一下。我的电脑有2个ipv6地址，我先用默认的参数添加了第一个解析记录，此时的解析是我的第一个地址。然后我让ddns-go使用第二个地址去添加，发现它只是新增了解析，并没有删掉原来的解析（每个配置了的域名都有两个解析）。所以后面我如果重启了电脑，那么之前的IP就没了，就需要一个一个删解析。所以想问一下有没有什么解决办法？

**Reply by xuanxuan1231** [reply]
![328800557c31d1991ad3747922ca010](https://github.com/jeessy2/ddns-go/assets/106209242/d0d9237f-4425-44cd-b34f-7ce795d90bf2)

**Reply by jeessy2** [reply]
参考https://github.com/cutepengpeng/DDNS-GO-IPV6
